### PR TITLE
Default sort order for targets and rollouts (ref #764)

### PR DESCRIPTION
**Pull request by AnbazhakanSubramaniam**
The default sort order has been change in order to show the newest
targets and rollouts are always on the top.

Signed-off-by: Anbazhakan Subramaniam <Anbazhakan.Subramaniam@in.bosch.com>

### CODE REVIEW

**Comment by schabdo** [comment]
@hawkbit-bot verify please

**Comment by schabdo** [comment]
Resolves https://github.com/eclipse/hawkbit/issues/764

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/SPUIDefinitions.java [hunk 1]
```diff
@@ -347,6 +347,7 @@
      * Sort order of column - created at in target table.
      */
     public static final Direction TARGET_TABLE_CREATE_AT_SORT_ORDER = Direction.ASC;
+    public static final Direction TARGET_TABLE_LASTMODIFIED_AT_SORT_ORDER = Direction.DESC;
```
Comment (laverman) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/SPUIDefinitions.java]: Please check if `TARGET_TABLE_CREATE_AT_SORT_ORDER` can be removed

Comment (AnbazhakanSubramaniam) [reply hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/SPUIDefinitions.java]: Thank you for the review comment!
The changes have been committed. 
Please review.

**Pull request merged**